WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewController.swift

```diff
@@ -72,7 +95,24 @@ private extension CustomerSelectorViewController {
         self.searchViewController = searchViewController
     }
 
-    func onCustomerTapped(_ customer: Customer) {}
+    func onCustomerTapped(_ customer: Customer) {
+        activityIndicator.startAnimating()
+        viewModel.onCustomerSelected(customer, onCompletion: { [weak self] result in
+            self?.activityIndicator.stopAnimating()
+
+            switch result {
+            case .success(()):
+                self?.dismiss(animated: true)
+            case .failure(_):
+                self?.showErrorNotice()
+            }
```

Comment:
We can remove those since we're not using the result's associated values:

```suggestion
            switch result {
            case .success:
                self?.dismiss(animated: true)
            case .failure:
                self?.showErrorNotice()
            }
```